content/en/graphing/guide/dashboard-lists-api-doc.md

```diff
@@ -0,0 +1,941 @@
+---
+title: Dashboard Lists API doc (v1)
+kind: guide
+aliases:
+  - /graphing/faq/dashboard-lists-api-doc
+---
+
+<div class="alert alert-danger">
+This endpoint is outdated. Use the <a href="https://docs.datadoghq.com/api/?lang=python#dashboard-lists">new Dashboard List endpoint version</a> instead.
+</div>
```

Comment:
Just like https://github.com/DataDog/documentation/pull/4587/files#r283440103 , I think we should have four of these messages above each endpoint with direct link to the `v2` endpoint. For example:

Use the **Get Items of a Dashboard List V2** endpoint (link to /api/?lang=python#get-items-of-a-dashboard-list)